astroquery/simbad/core.py

```diff
@@ -936,6 +1006,243 @@ def query_objectids_async(self, object_name, *, cache=True,
 
         return response
 
+    def tables(self, get_adql=False):
```

Comment:
In general I'm more in favour of using more descriptive method names. We don't have a strong precedent yet, but `list_tables` or `get_tables` may work.

Reply:
Much clearer, thanks a lot for the suggestion. I chose to use `list_tables`, `list_columns` and `list_linked_tables`